exchanges/refocus/src/refocusExchange.ts

```diff
@@ -0,0 +1,38 @@
+import { pipe, tap } from 'wonka';
+import { Exchange, Operation } from '@urql/core';
+
+export const refocusExchange = (): Exchange => {
+  return ({ client, forward }) => ops$ => {
+    const watchedOperations = new Map<number, Operation>();
+    const observedOperations = new Map<number, number>();
+
+    window.addEventListener('focus', () => {
+      watchedOperations.forEach(op => {
+        client.reexecuteOperation(
+          client.createRequestOperation(
+            'query',
+            { key: op.key, query: op.query, variables: op.variables },
```

Comment:
Fun fact, I think `Operation` does extend `GraphQLRequest`, so you can pass the entire `op` here to save some bytes